### PR TITLE
Added changes to the output script

### PR DESCRIPTION
**Pull request by k1502606**
I have changes the output script of the common diff.

### CODE REVIEW

File: Differencer/src/main.rs [hunk 1]
```diff
@@ -174,19 +174,34 @@ fn printAllVectorOfVector(input:Vec<Vec<String>>){
...
+	
+ 	let  mut test1 = lines_from_file("C:/Users/Abhishek Shankar/Desktop/Uni/Project/Rust Project/Rust Code/Differencer/src/test1.txt");
+ 	let mut test2 = lines_from_file("C:/Users/Abhishek Shankar/Desktop/Uni/Project/Rust Project/Rust Code/Differencer/src/test2.txt");
 
- 	let  mut test1 = lines_from_file("C:/Users/Abhishek Shankar/Desktop/Rust/Differencer/src/test1.txt");
- 	let mut test2 = lines_from_file("C:/Users/Abhishek Shankar/Desktop/Rust/Differencer/src/test2.txt");
-
-
+    // let mut array1:Vec<String> = multiLineString_ToVector_SplitBySpace("A B C");
+    // let mut array2:Vec<String> = multiLineString_ToVector_SplitBySpace("A X B C");
```
Comment (snim2) [inline Differencer/src/main.rs]: Remove commented code...

File: Differencer/src/main.rs [hunk 2]
```diff
@@ -535,12 +550,59 @@ fn diff(string1: &mut Vec<String>, string2: &mut Vec<String>, mut i: u32, mut j:
 }
 
 fn print_edit_sequence(vector:Vec<ReturnValue>,input1:Vec<String>,input2:Vec<String>){
-    println!("------------------------------------ RESULTS STR ------------------------------------");
-
+    // println!("------------------------------------ RESULTS STR ------------------------------------");
```
Comment (snim2) [inline Differencer/src/main.rs]: Maybe remove all this!

File: Differencer/src/main.rs [hunk 3]
```diff
@@ -535,12 +550,59 @@ fn diff(string1: &mut Vec<String>, string2: &mut Vec<String>, mut i: u32, mut j:
 }
 
 fn print_edit_sequence(vector:Vec<ReturnValue>,input1:Vec<String>,input2:Vec<String>){
```
Comment (snim2) [inline Differencer/src/main.rs]: Maybe we can remove this function if it just calls another function.

**Comment by snim2** [comment]
Looks good! Let's squash

**Comment by k1502606** [comment]
I have squashed the second commit.

**Pull request merged**